OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -37,9 +37,307 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="6ra" to="46ra"/>
+                            <title type="complete" ref="LIT1828Mahale"/>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="6ra" to="8va"/>
+                                <title type="complete" ref="LIT1828Mahale#Moses1"/>
+                                
+                                <incipit xml:lang="gez">
+                                    <locus from="6ra" to="6rb"/>
+                                    <pb n="6r"/><cb n="a"/>
+                                    <seg type="inscriptio"><hi rend="rubric">ማኅልየ፡ ነቢያት፡</hi> ወጸሎቱ፡ ለሙሴ፡</seg>
+                                    <hi rend="rubric">፩ ንሴብሖ፡ ለእግ</hi>ዚአብሔር፡ ስቡሐ፡ ዘተሰብሐ<hi rend="rubric" rendition="#partialRubric">፨</hi> ፈረሰ፡ ወመስተፅዕነ፡ ወረወ፡
+                                    ውስተ፡ ባሕር<hi rend="rubric" rendition="#partialRubric">፨</hi> ረዳኤ፡ ወመሰወረ፡ <hi rend="rubric">ኮነኒ፡
+                                        ኦአድኅኖትየ<hi rend="rubric" rendition="#partialRubric">፤</hi> ው</hi><gap reason="lost" unit="chars" extent="2"/>፡ አምላኪየ፡
```

Comment:
I copy herethe issue concering the use of @extent and @quantity in case also @Massimo encoded it wrongly as Thea and me 

https://github.com/BetaMasaheft/Documentation/issues/1519